identity-service/src/audiusLibsInstance.js

```diff
@@ -35,4 +33,16 @@ async function initAudiusLibs () {
   return audiusInstance
 }
 
-module.exports = initAudiusLibs
+class AudiusLibsWrapper {
+  constructor () {
```

Comment:
you can't call an async function in a constructor. other async non-constructor functions can be made async in the class

Reply:
Thanks, updated to no longer call an async function and uses `null` instead.